.circleci/config.yml

```diff
@@ -2253,7 +2241,7 @@ jobs:
             cd /swirlds-platform;
             sed -i -e 's/github.com/services-jrs-regression/g' .gitmodules;
             git submodule update --init --recursive --checkout;
-            cd regression;
+            cd regression; git checkout 1207-M-store-logs-to-gs;
```

Comment:
Restore this line before merge.

Reply:
Thanks @ljianghedera . As I mentioned in the PR description will merge it once regression PR is merged tomorrow.

Reply:
@ljianghedera @qnswirlds Reverted the regression branch.